wrappers/java/src/test/java/org/hyperledger/indy/sdk/IndyIntegrationTest.java

```diff
@@ -56,28 +56,20 @@
 	protected String XYZ_SCHEMA_ATTRIBUTES = "[\"status\", \"period\"]";
 	protected String REVOC_REG_TYPE = "CL_ACCUM";
 	protected String SIGNATURE_TYPE = "CL";
-	protected String TAILS_WRITER_CONFIG = new JSONObject(String.format("{\"base_dir\":\"%s\", \"uri_pattern\":\"\"}", getIndyHomePath("tails")).replace('\\', '/')).toString();
+	protected String TAILS_WRITER_CONFIG =
+			"{ \"base_dir\":" +  getIndyHomePath("tails").replace('\\', '/') + ", \"uri_pattern\":\"\"}";
 	protected String REV_CRED_DEF_CONFIG = "{\"support_revocation\":true}";
 	protected String GVT_CRED_VALUES = "{\n" +
 			"        \"sex\": {\"raw\": \"male\", \"encoded\": \"5944657099558967239210949258394887428692050081607692519917050\"},\n" +
 			"        \"name\": {\"raw\": \"Alex\", \"encoded\": \"1139481716457488690172217916278103335\"},\n" +
 			"        \"height\": {\"raw\": \"175\", \"encoded\": \"175\"},\n" +
 			"        \"age\": {\"raw\": \"28\", \"encoded\": \"28\"}\n" +
 			"    }";
-	protected static final String WALLET_CONFIG =
-			new JSONObject()
-					.put("id", WALLET)
-					.put("storage_type", TYPE)
-					.toString();
-	protected static final String WALLET_CREDENTIALS =
-			new JSONObject()
-					.put("key", "key")
-					.toString();
-	protected static final String PLUGGED_WALLET_CONFIG =
-			new JSONObject()
-					.put("id", WALLET)
-					.put("storage_type", "unknown_type")
-					.toString();
+	protected static final String WALLET_CONFIG = "{ \"id\":" + WALLET + ", \"storage_type\":" + TYPE + '}';
```

Comment:
Please check it. Seems we need to manually add quotes.
For me, WALLET_CONFIG will be `{ "id":Wallet1, "storage_type":default}`.